src/main.rs

```diff
@@ -154,8 +162,11 @@ fn main() {
 
   let rc = match config::Config::load() {
     Ok(config) => {
+      use crossterm::tty::IsTty as _;
+
+      let is_tty = stdout.is_tty();
```

Comment:
I honestly do not think we should pull in a dependency for this functionality. Even more so given how infrequently this command can be expected to be used and how much %#&(*$ this thing pulls in transitively. Let's use `libc::isatty` on `stdout.as_raw_fd()` if we are on Unix and just false everywhere else.

Reply:
True, but crossterm is also used to clear the current line when writing a progress bar, see below.